modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -180,93 +182,137 @@ const SidebarPanelInfoView = ({
-							{Liferay.Language.get('categories')}
-						</h5>
-
-						<p>
-							{categories.map((category) => (
-								<ClayLabel
-									displayType="secondary"
-									key={category}
-									large
-								>
-									{category}
-								</ClayLabel>
-							))}
-						</p>
-					</div>
-				)}
-
-				{!!tags.length && (
-					<div className="c-mb-4 sidebar-dl sidebar-section">
-						<h5 className="c-mb-1 font-weight-semi-bold">
-							{Liferay.Language.get('tags')}
-						</h5>
-
-						<p>
-							{tags.map((tag) => (
-								<ClayLabel
-									displayType="secondary"
-									key={tag}
-									large
-								>
-									{tag}
-								</ClayLabel>
-							))}
-						</p>
-					</div>
-				)}
-
-				<h6 className="font-weight-semi-bold sidebar-section-subtitle text-uppercase">
-					{Liferay.Language.get('details')}
-				</h6>
-
-				{documentIsAFile && (
-					<div className="sidebar-section">
-						<h5 className="c-mb-1 font-weight-semi-bold">
-							{Liferay.Language.get('extension')}
-						</h5>
-
-						<p className="text-secondary">{extension}</p>
+				<ClayPanel.Group className="panel-group-flush panel-group-sm">
+					{showTaxonomies && (
+						<ClayPanel
+							collapsable
+							displayTitle={
+								<span className="c-inner" tabIndex="-1">
+									<ClayPanel.Title>
+										<ClayLayout.ContentRow>
+											<ClayLayout.ContentCol
+												className="align-self-center panel-title"
+												expand
+											>
+												{Liferay.Language.get(
+													'categorization'
+												)}
+											</ClayLayout.ContentCol>
+										</ClayLayout.ContentRow>
+									</ClayPanel.Title>
+								</span>
+							}
+							displayType="unstyled"
+							showCollapseIcon={true}
+						>
+							<ClayPanel.Body>
+								{!!categories.length && (
+									<div className="c-mb-4 sidebar-dl sidebar-section">
+										<h5 className="c-mb-1 font-weight-semi-bold">
+											{Liferay.Language.get('categories')}
+										</h5>
+
+										<p>
+											{categories.map((category) => (
+												<ClayLabel
+													displayType="secondary"
+													key={category}
+													large
+												>
+													{category}
+												</ClayLabel>
+											))}
+										</p>
+									</div>
+								)}
+
+								{!!tags.length && (
+									<div className="c-mb-4 sidebar-dl sidebar-section">
+										<h5 className="c-mb-1 font-weight-semi-bold">
+											{Liferay.Language.get('tags')}
+										</h5>
+
+										<p>
+											{tags.map((tag) => (
+												<ClayLabel
+													displayType="secondary"
+													key={tag}
+													large
+												>
+													{tag}
+												</ClayLabel>
+											))}
+										</p>
+									</div>
+								)}
+							</ClayPanel.Body>
+						</ClayPanel>
+					)}
 
-						<h5 className="c-mb-1 font-weight-semi-bold">
-							{Liferay.Language.get('size')}
-						</h5>
+					<ClayPanel
```

Comment:
And maybe in the same way than the previous comment ¿could be a good idea to move all the ClayPanel to another component? We could use `children`prop for the content slot, but the rest is the same in all cases...
What do you think?

Reply:
Hey Beltran, I think that it makes sense to move the Panel to a new Component 👍